tests/clickhouse/translators/snuba/test_transaction_translations.py

```diff
@@ -0,0 +1,235 @@
+import pytest
```

Comment:
Could you please move this in tests/datasets ? This test is meaningful for a specific dataset, and not relevant to the translation framework.